toolchest_client/query.py

```diff
@@ -252,23 +292,23 @@ def _download(self, output_path):
 
         download_signed_url = self._get_download_signed_url()
 
-        self._update_status(Status.TRANSFERRING_TO_CLIENT.value)
+        self._update_status(Status.TRANSFERRING_TO_CLIENT)
 
         # Dowloads output by sending a GET request.
         with requests.get(download_signed_url, stream=True) as r:
             # Validates reponse of GET request.
             try:
                 r.raise_for_status()
             except HTTPError:
-                print("Output download failed.")
-                raise
+                self._raise_for_failed_client("Output download failed.")
```

Comment:
I don't think there's a case outside the request itself failing. (If the request fails, then we can't download it, so we would still probably want to update to `failed`.) 

This might be a good case to use the "failed but retrying" status/procedure that we briefly discussed.

Reply:
👍